src/restyleFunctions.ts

```diff
@@ -44,6 +47,9 @@ const spacingPropertiesShorthand = {
   py: 'paddingVertical',
   ps: 'paddingStart',
   pe: 'paddingEnd',
+  g: 'gap',
+  rG: 'rowGap',
+  cG: 'columnGap',
```

Comment:
Wasn't it better to keep abbreviations consistent?`rg` and `cg`?

Reply:
You're right, I don't use shorthands and didn't notice they are in lowercase